hedyorg/hedy
# [CHORE] Fix setup.cfg by extending it with specific autopep8 config

**Description**

This edits the setup.cfg to prevent autopep8 from trying to inherit per-file-ignores from the flake8 config as autopep8 does not support this (yet, anyone up for a coding challenge?) 

It also updates a ton of files to be pep8-compliant again as these slipped through since the last time I ran a formatter. 

Flake8 also found a test that didn't do anything in test_level_07.py; please verify if I added the assert correctly. The unit tests are still running but I'm assuming it should be all right. 

**Fixes _issue or discussion number_**

Related to #3377

**How to test**

run autopep8 on the entire repo with this setup.cfg, you'll see that it skips the tests repo
